packages/diagram-ui/src/features/undo-redo/remoteUndoRedoKeyListener.ts

```diff
@@ -0,0 +1,20 @@
+import { RemoteRedoAction, RemoteUndoAction } from "@hylimo/diagram-protocol";
+import { isCtrlOrCmd, isMac, KeyListener, SModelElementImpl } from "sprotty";
+import { Action } from "sprotty-protocol";
+
+/**
+ * Key listener for remote undo and redo actions
+ */
+export class RemoteUndoRedoKeyListener extends KeyListener {
+    override keyDown(element: SModelElementImpl, event: KeyboardEvent): Action[] {
+        if (isCtrlOrCmd(event)) {
+            if ((!isMac() && event.key.toLowerCase() == "y") || (event.shiftKey && event.key.toLowerCase() == "z")) {
```

Comment:
I would remove the `isMac` call: Give all users the same shortcuts.
Since I know some websites offer Ctrl+y and some Ctrl+Shift+Z, why not simply offer both?

Additionally, What is our proposed behavior on Ctrl+Shift+y?

Reply:
current behavior is aligned with the original sprotty code
I don't really care because I'm not a mac user, and different programs seem to handle this differently
Ctrl + Shift + Y does the same as Ctrl + Y, however I can change that if you want me to (although this of course opens the questions, what do other modifiers than do)

Reply:
removed mac special behavior with ddded50313b0446facc10b30f314953566d3b02e

Reply:
Yes, I do prefer keeping the behavior equal everywhere,
and yes, this question was meant symbolically for all other modifiers as well

Reply:
I personally would allow other modifiers as long as they do not conflict with another command, but I also don't really care